### ISSUE TITLE
Sorry, an error occurred while trying to access this service.

### ISSUE THREAD

**Issue by snailCorpse** [OfficeDev/microsoft-teams-apps-company-communicator#338]
After re-deploying to the latest version from the ground up, I'm now getting the error:
"Sorry, an error occurred while trying to access this service."

Looking at the errors in the console:
Failed to load resource: the server responded with a status of 401 (Unauthorized) 

On the services
api/draftnotifications 
api/sentnofications

**Comment by tiagomiotto** [comment]
Same issue here

**Comment by snailCorpse** [comment]
The developers don't answer, unfortunately so I doubt we will get an answer. I'm just hoping that someone will release a version that works. The only version I got to work was version 2, since I updated the communicator it has just been issues.

Deploying from scratch does not help. There is something fundamentally wrong with the architecture and it only works for some subscriptions and some users, perhaps its region related or features not available to all Azure subscriptions. I don't know
